.travis/before-install-osx.sh

```diff
@@ -2,12 +2,22 @@
 
 export HOMEBREW_NO_ANALYTICS=1
 
-brew install libsndfile || brew install libsndfile || exit 1
+if $UPDATE_HOMEBREW; then
+    #run update first so that possible update errors won't hold up package installation
+    brew update
```

Comment:
`brew update --preinstall` is faster and does the same thing

Reply:
Also should I disable update for legacy build or leave it in?